twinkle-app/twinkle.dal.repositories/src/main/java/com/github/donkeyrit/twinkle/dal/repositories/UserRepositoryImpl.java

```diff
@@ -60,4 +60,12 @@ public void insert(User user)
         session.persist(user);
         session.getTransaction().commit();
     }
+
+    @Override
+    public void remove(User user)
```

Comment:
Try to use `BaseCrudRepository` because this class implement removing feature.

Reply:
Or try to merge with a new master firstly)

Reply:
Done